api/tests/team/unit/application/admin-member/admin-member.route.test.js

```diff
@@ -234,4 +234,61 @@ describe('Unit | Team | Application | Router | admin-member', function () {
       expect(statusCode).to.equal(403);
     });
   });
+
+  describe('PUT /api/admin/admin-members/{id}/deactivate', function () {
+    const method = 'PUT';
+    const url = '/api/admin/admin-members/1/deactivate';
+
+    describe('When user has role "SUPER_ADMIN"', function () {
+      it('should return a response with an HTTP status code 200', async function () {
+        // given
+        sinon
+          .stub(adminMemberController, 'deactivateAdminMember')
+          .callsFake((request, h) => h.response('ok').code(204));
+        sinon.stub(securityPreHandlers, 'checkAdminMemberHasRoleSuperAdmin').returns(true);
+        const httpTestServer = new HttpTestServer();
+        await httpTestServer.register(teamRoutes[0]);
+
+        // when
+        const { statusCode } = await httpTestServer.request(method, url);
+
+        // then
+        // dim
```

Comment:
ça veut dire quoi //dim ? 🧐

Reply:
Development Iterative and Modelized (ou bien Dimitri je sais plus)